docassemble/juvenilesealing/data/questions/juvenile-sealing-intro-10.yml

```diff
@@ -180,11 +191,26 @@ subquestion: |
   Once you seal your records, you'll have to unseal them to get
   certified copies. That can sometimes take a while.
 
+  ### What are docket sheets?
+
+  ${docket_sheets_blurb} Docket
+  sheets are papers from the courthouse that say
+  what happened in a case every time you went
+  to court for that case.
+
+  ### What's a certified copy of docket sheets?
+
+  A certified copy of docket sheets
+  will have a special mark on it that looks like the
+  courthouse's seal. It'll either be a colored stamp or
+  an area where the paper is raised up in the shape of the
+  seal.
+
   Do you want to get certified copies of your docket sheets before
```

Comment:
So I think this question was originally meant to be at the end of the process of learning about your docket sheets. Options:
1. Keep these questions here to allow the person to resume the interview and skip the rest of the text.
1. Change this to our 'keep going' to force them to see it all.
1. To consider separately from the others: Put questions like this at the end of the docket sheet informational section to allow the user the options to exit the interview or to keep filling out the form.

Reply:
I don't have an opinion between these three options. If you have one, I can go with that. Otherwise, we can discuss it on Tuesday.

Reply:
Tuesday sounds good.

Reply:
Notes from meeting - this is what we want: 

Page 1: why docket sheets are important and what they are 
Option 1: continue to where you were in the quiz 
Option 2: Go to page 2
~Option 3: Stop the application and come back later~ 

Page 2: how to get docket sheets 
Option 1: continue to where you were in the quiz 
Option 3: Stop the application and come back later